### DISCUSSION TITLE
Use Whisper With A Microphone

### DISCUSSION THREAD

**Discussion by mallorbc** [openai/whisper#75] [Show and tell]
I have created a repo that allows one to use Whisper with a microphone in real time. 
https://github.com/mallorbc/whisper_mic

**Comment by nn81** [comment]
Thanks for sharing. There should be a section on the Whisper README to list these extensions.

It is exactly what I've been looking to automate actions with voice. Thank you.

**Reply by UmangRajpara13** [reply]
@nn81 perhaps you might also like this [repo](https://github.com/voyagingstar/able). it records user input with mic, transcribes it and then runs user defined voice commands(stored in a json dictionary).  also you can control other apps like VS Code by writing an extension.
